### PR TITLE
UX: Mobile styling.

### PR DESCRIPTION
**Pull request by tgxworld**
#### Before 

Screenshot from 2022-02-07 15-23-21

#### After

Screenshot from 2022-02-07 15-23-07

#### Before

Screenshot from 2022-02-07 15-27-35

#### After

Screenshot from 2022-02-07 15-28-02

### CODE REVIEW

File: assets/stylesheets/common/question-answer.scss [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+  .qa-comments-menu-show-more,
+  .qa-comments-menu-add-link {
+    display: inline;
+  }
+}
+
+.qa-post {
+  .qa-post-list {
+    box-shadow: shadow("card");
```
Comment (nattsw) [inline assets/stylesheets/common/question-answer.scss:118]: Btw, is this valid?

Comment (tgxworld) [reply assets/stylesheets/common/question-answer.scss:118]: Yea this is a SASS function in core: https://github.com/discourse/discourse/blob/357186ab7ebd100c38d4776e9e05e9d2d988f570/app/assets/stylesheets/common/foundation/variables.scss#L147-L149

**Review by nattsw — APPROVED** [review]

**Comment by nattsw** [comment]
Btw, it seems now that everything is in the "common" stylesheet. You may want to consider component specific stylesheets.

**Comment by tgxworld** [comment]
Will do :) We kind of don't have a convention on stylesheets so usually it is anything goes xD Maybe I should create a topic to discuss it. Thank you for the review :+1:

**Pull request merged**